src/Constants.js

```diff
@@ -28,6 +46,11 @@ export const skillsOptions = [
   { label: "VueJs", value: "VueJs" },
 ];
 
+export const ROUTES = {
```

Comment:
please add a separete files for routes path constants
e.g - 
export const PROFILE_API_URL = '/api/profile/';